pkg/hooks/monitor.go

```diff
@@ -2,9 +2,8 @@ package hooks
 
 import (
 	"context"
-	"os"
-	"path/filepath"
 
+	current "github.com/containers/libpod/pkg/hooks/1.0.0"
```

Comment:
suggest `libpodhook` over `current`, unless @mheon has a better suggestion.

Reply:
I'd call it `hooks-cur` or `hooks-current`

Reply:
Never mind, I see current is in the package.